apps/src/code-studio/header.js

```diff
@@ -176,8 +176,7 @@ function shareProject() {
           appType={appType}
           onClickPopup={popupWindow}
           // TODO: Can I not proliferate the use of global references to Applab somehow?
```

Comment:
Think we'll ever address this TODO?  Can we remove it if it doesn't represent an independent action item?